pkg/collector/collector.go

```diff
@@ -42,6 +44,11 @@ const (
 	defaultForwardingUpTimer = 5
 )
 
+var supportedMembersTypes = []string{
```

Comment:
can you update readme with supported and unsupported group member types? there is already a separate page for such documentation, where this should be added.

Reply:
may open a separate issue for this.